packages/sakuli-legacy/src/context/legacy-lifecycle-hooks.class.ts

```diff
@@ -138,6 +125,26 @@ export class LegacyLifecycleHooks implements TestExecutionLifecycleHooks {
     });
   }
 
+  private async createDriver(project: Project) {
+    this.driver = createDriverFromProject(project, this.builder);
+    await this.driver.manage().window().maximize();
```

Comment:
I'd add some debug logging to the browser creation just to indicate that the instance was newly created.